Userland/Utilities/strace.cpp

```diff
@@ -210,11 +210,7 @@ END_VALUES_TO_NAMES()
 
 static int g_pid = -1;
 
-#if ARCH(I386)
-using syscall_arg_t = u32;
-#elif ARCH(X86_64) || ARCH(AARCH64)
```

Comment:
shall we keep the double-check two handled  64-bit architectures here and error in the opposite case? 🤔